spec/components_spec.rb

```diff
@@ -355,8 +355,8 @@
 
 RSpec.describe "raw" do
   it 'creates a wrapper that ignores anything inside' do
-    input = "<raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw>"
-    expected = "<<LCG Program\TG LCG Coupon Code Default='246996'>>"
+    input = "<body><raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw></body>"
+    expected = "<?xml version=\"1.0\"?>\n<body><<LCG ProgramTG LCG Coupon Code Default='246996'>></body>\n"
```

Comment:
I changed this test because I was having an issue where Nokogiri was expecting a root node. I figured it was alright to assume at least a <body> tag will be present. 

When only `<raw>` was used it was passing a string to Nokogiri like `###RAW0###` which was ultimately turning into a blank Nokogiri document. @kball please confirm this is an appropriate change to the tests.